server/lib/nl/fulfillment/ranking_across_vars.py

```diff
@@ -75,7 +75,8 @@ def _populate_cb(state: PopulateState, chart_vars: ChartVars,
     return False
 
   # Ranking among peer group of SVs.
-  chart_vars.svs = utils.ranked_svs_for_place(places[0].dcid, chart_vars.svs,
-                                              state.ranking_types[0])
+  chart_vars.svs = utils.rank_svs_by_latest_value(places[0].dcid,
```

Comment:
does it matter if there are multiple places here? Maybe add a comment on implication of selecting the first one.

Reply:
The API has multiple places (because it handles place comparison).